stellarphot/photometry/tests/test_photometry.py

```diff
@@ -514,14 +535,26 @@ def test_photometry_on_directory():
 
         expected_deviation = np.pi * aperture**2 * noise_dev
 
-        # We could require that the result be within some reasonable
-        # number of those expected variations or we could count up the
-        # actual number of background counts at each of the source
-        # positions.
-
-        # Here we just check whether any difference is consistent with
-        # less than the expected one sigma deviation.
-        assert np.abs(expected_flux - obs_avg_net_cnts) < expected_deviation
+        # We have two cases to consider: use_coordinates="sky" and
+        # use_coordinates="pixel". In the former case, the expected
+        # result is the test below. In the latter case, the expected
+        # result is that either obs_avg_net_cnts is nan or the difference
+        # is bigger than the expected_deviation.
```

Comment:
Am I misreading this?  You say you expect the different is BIGGER than the expected_deviation, but below (line 556) you assert it is smaller, no?

Also, just for my sake, can you clarify why the noise will be different in sky coords versus pixel coord?

Reply:
Two things

+ I rearranged the comments to make clear what the passing condition is in each case.
+ I think there is a bug somewhere, either
    - In `list_of_image` or in the shifting stuff that calls. The reason I say that is I think we should be able to use either pixel or sky coordinates and have this pass. In practice, the image first image is fine but the second gets negative net counts and bad fwhms.
    - Maybe in `multi_image_photometry` since that treats the first image different than the rest.

I'll track down what exactly the error is later this morning, I hope.

Reply:
> You say you expect the different is BIGGER than the expected_deviation, but below (line 556) you assert it is smaller, no?

oops, now fixed